cmd/tsdbrelay/main.go

```diff
@@ -14,6 +15,8 @@ import (
 	"os"
 	"strings"
 
+	version "bosun.org/_version"
```

Comment:
@captncraig I'm curious why I had to put an alias here. Locally using go 1.6.3 it can't find the reference to version package and gives an error/removes the import if I try to use a default import or even the _ import. Scollector/Bosun seem to work fine without the alias.